ghost/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: ghost_web_1
+      APP_PORT: 2368
+
+  db:
+    image: mysql:9.2@sha256:d56d039139a7f3b71f6d1c9f07ca4ee9f977b0fca13acdd27a1b13bfd4a4e3be

```

According to the ghost documentation, only version 8 of mysql is supported. Lets use 8.4 (LTS).
```suggestion
    image: mysql:8.4.4@sha256:339598fe066c73daa4120f2d9dfefd3146c3519cd324748e1d3f9d2b6c71d10b

```